Complications/Wearable/src/main/java/com/example/android/wearable/wear/complications/ShortTextDataSourceService.kt

```diff
@@ -130,4 +133,6 @@ class ShortTextDataSourceService : SuspendingComplicationDataSourceService() {
         TEXT_WITH_TITLE,
         TEXT_WITH_ICON_AND_TITLE,
     }
+
+    private fun safeText(@StringRes resId: Int): String = getText(resId).toString()
```

Comment:
What is safe about this given `getText()` can't be null and CharSequence is accepted by the builders?